### PR TITLE
Upgrade quill to v2.0.2

### PR DESCRIPTION
**Pull request by jonatanschroeder**
As per https://quilljs.com/docs/upgrading-to-2-0, some adjustments were necessary:

* Update call to `convert`;
* Embed class has a different method of importing;
* `dist/quill.js` is now the minified version, so `dist/quill.min.js` no longer exists.

This update has been tested locally and works as expected. It may break any custom element that depends on the previous version of quill, though.

### CODE REVIEW

File: apps/prairielearn/elements/pl-rich-text-editor/pl-rich-text-editor.js [hunk 1]
```diff
@@ -45,10 +45,10 @@
     if (contents && renderer) contents = renderer.makeHtml(contents);
     contents = rtePurify.sanitize(contents, rtePurifyConfig);
 
-    quill.setContents(quill.clipboard.convert(contents));
+    quill.setContents(quill.clipboard.convert({ html: contents }));
```
Comment (jonatanschroeder) [inline apps/prairielearn/elements/pl-rich-text-editor/pl-rich-text-editor.js:48]: https://quilljs.com/docs/upgrading-to-2-0#clipboard

File: apps/prairielearn/elements/pl-rich-text-editor/pl-rich-text-editor.js [hunk 2]
```diff
@@ -45,10 +45,10 @@
...
 
     quill.on('text-change', function () {
-      let contents = rtePurify.sanitize(quill.root.innerHTML, rtePurifyConfig);
+      let contents = rtePurify.sanitize(quill.getSemanticHTML(), rtePurifyConfig);
```
Comment (jonatanschroeder) [inline apps/prairielearn/elements/pl-rich-text-editor/pl-rich-text-editor.js:51]: https://quilljs.com/docs/api#getsemantichtml

File: apps/prairielearn/elements/pl-rich-text-editor/pl-rich-text-editor.js [hunk 3]
```diff
@@ -63,7 +63,7 @@
 
   // Override default implementation of 'formula'
 
-  var Embed = Quill.imports.parchment.Embed;
+  var Embed = Quill.import('blots/embed');
```
Comment (jonatanschroeder) [inline apps/prairielearn/elements/pl-rich-text-editor/pl-rich-text-editor.js:66]: https://quilljs.com/docs/upgrading-to-2-0#parchment

**Review by nwalters512 — APPROVED** [review]
I tested this locally and things appear to work!

**Pull request merged**